website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -346,10 +362,18 @@ If you are encountering issues launching the GUI at this point, please head to t
 
 ## Exported Data
 
-Data from the board will be recorded into a **Brainflow-RAW_date_time** CSV file and **OpenBCI-RAW_date_time** CSV file in txt format. All data saved to these files are unfiltered. **These recordings are stored in `User/Documents/OpenBCI_GUI/Recordings`**. Each GUI session will have its own directory titled **OpenBCISession_date_time**. Every time you start and stop the data stream, the GUI creates new recording files within that session folder. The picture below shows three recordings within one GUI session. The BrainFlow CSV files are generated by the BrainFlow Streamer feature when set to stream to file. This feature can also be used to send raw data over a network. By default, BrainFlow streamer saves data to this folder.
+Data from the board will be recorded into a **BrainFlow-RAW_date_time** CSV file and **OpenBCI-RAW_date_time** CSV file in txt format. All data saved to these files are unfiltered. **These recordings are stored in `User/Documents/OpenBCI_GUI/Recordings`**. Each GUI session will have its own directory titled **OpenBCISession_date_time**. Every time you start and stop the data stream, the GUI creates new recording files within that session folder. The picture below shows three recordings within one GUI session. The BrainFlow CSV files are generated by the BrainFlow Streamer feature when set to stream to file. This feature can also be used to send raw data over a network. By default, BrainFlow streamer saves data to this folder.
```

Comment:
Why is "These records are stored in" bolded?

Reply:
It's for the people that need help finding where the data is stored. We get support tickets on this. The primary reason for the creation of this section is based around support tickets I have participated in. That's why I chose to add a little more info in this PR.

Reply:
In that case, I would recommend putting the sentence at the top of the section (under the Exported Data header) on it's own line. Even better, make it an ::info or similar. As it stands it looks weird (to me it looked like a mistake) buried in the paragraph.

Reply:
Agreed. Thanks for the review.

Reply:
Better yet -- own line and with a linkable heading.